.travis.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         -e CI=true \
         --env-file .env.example \
         app/test /bin/bash -c "tail -f /dev/null"
-      docker exec test-container rake
+      docker exec test-container bundle exec rake
```

Comment:
In theory since we are only installing the dependencies we need in a completely clean environment we should be able to remove all instance of bundle exec. 

If we are prompted to use it to run the command then we have probably installed 2 different versions of the same package somewhere so it would be useful to know that.

I propose that this commit flips direction to remove all cases for consistency (though it was probably me that added it for brakeman, sorry!)

Reply:
What about different version of dependancies? Will it handle that case - does that case even exsist? I'd like to understand this more.

Reply:
Also see https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/296#discussion_r400844590

Reply:
My non-understanding grows wider with this bundler business! 

I can call `bundle` and `rake` as they are both in the path at `/usr/local/bundle/bin/` so any call to `rake` can be sans `bundle exec` - is this sound as the container only has one single version of rake and one 'global' bundle?

I also have a binstub for rake in `/app/bin` but that is part of the code so I don't use it.

`rails` on the other hand is a gem, so must be run via `bundle exec`?

Reply:
☝️ I think this is why I run `rails` everywhere!